src/app/CASESessionManager.h

```diff
@@ -156,6 +162,8 @@ class CASESessionManager : public OperationalSessionReleaseDelegate, public Sess
      */
     CHIP_ERROR GetPeerAddress(const ScopedNodeId & peerId, Transport::PeerAddress & addr);
 
+    CHIP_ERROR GetPeerAddressForLargePayloadSession(const ScopedNodeId & peerId, Transport::PeerAddress & addr);
```

Comment:
Why wouldn't we just have a TransportPayloadCapability argument to GetPeerAddress, either via defaulted arg or via overload?

Reply:
Added as a defaulted arg to GetPeerAddress().